src/App.jsx

```diff
@@ -14,45 +16,103 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const ALPHABET_SORT_FIELD = 'alphabet';
+const LENGTH_SORT_FIELD = 'length';
+const DEFAULT_SORT_FIELD_VALUE = '';
+
+function getPreparedGoods(goods, { sortField, isReversed }) {
```

Comment:
Why do you pass the second and third arguments as an one object? What is the point in this?